modules/creating-the-vsphere-windows-vm-golden-image.adoc

```diff
@@ -68,6 +68,14 @@ The public key used in the instructions must correspond to the private key you c
 
 . Install the `docker` container runtime on your Windows VM following the link:https://docs.microsoft.com/en-us/virtualization/windowscontainers/quick-start/set-up-environment?tabs=Windows-Server[Microsoft documentation].
 
+. Enable the `Containers` feature on the Windows OS. This feature is disabled by default on Windows Server 2022. 
+
++
+[source,posh]
+----
+PS C:\> Install-WindowsFeature -Name Containers
+----
+
```

Comment:
@mburke5678 
this is actually outdated information, as we do this for the user, instead of having them do it.

I noticed that the step above
```
 . Install the `docker` container runtime on your Windows VM following the link:https://docs.microsoft.com/en-us/virtualization/windowscontainers/quick-start/set-up-environment?tabs=Windows-Server[Microsoft documentation].
 ```
 is also outdated. That should be removed, as we no longer have docker as a requirement for WMCO.  This is true for both 4.11 and 4.12

Reply:
To be clear, I am saying
1) this addition should not be added
2) the instructions to install docker should be removed